Server/Models/fbContext.cs

```diff
@@ -4,7 +4,7 @@
 
 #nullable disable
 
-namespace Server.Models
+namespace PostApp.Models
 {
     public partial class fbContext : DbContext
```

Comment:
Change to PostAppContext

Reply:
changed the name of context to PostAppContext